assets/js/buttonMenu.js

```diff
@@ -6,4 +6,4 @@ $(() => {
     menuClasses: 'moj-button-menu__wrapper--right',
     buttonClasses: "govuk-!-margin-right-0",
   })
-})
\ No newline at end of file
+})
```

Comment:
We're using ESlint and our config has a rule for having a new line at the end of the file, not sure how this one slipped through :)

Reply:
Probably because it's a `.js` file and the `.eslintrc.js` file is only configured to lint `['*.test.ts', '*.spec.ts', '*.test.js', '*.spec.js']` files.

Reply:
actually I think I was looking at the config from another project so I dunno 😅